src/python/pants/jvm/resolve/coursier_fetch.py

```diff
@@ -312,6 +344,47 @@ def classpath_dest_filename(coord: str, src_filename: str) -> str:
     return f"{dest_name}{ext}"
 
 
+@dataclass(frozen=True)
+class ArtifactRequirementsWithLocalFiles:
+    # omg tidy up this name. it's bad
```

Comment:
Would be good to clean this up: https://www.pantsbuild.org/docs/style-guide#style

Reply:
oops, tidied up the name and forgot to delete the comment Fixing.